redbot/core/events.py

```diff
@@ -99,10 +100,96 @@ async def on_ready():
                     "Outdated version! {} is available "
                     "but you're using {}".format(data["info"]["version"], red_version)
                 )
-                outdated_red_message = (
+                outdated_red_message = _(
                     "Your Red instance is out of date! {} is the current "
                     "version, however you are using {}!"
                 ).format(data["info"]["version"], red_version)
+                requires_python = data["info"]["requires_python"]
+                current_python = platform.python_version()
+                extra_update = _(
+                    "\n\nWhile the following should work in most scenarios as it is "
+                    "based on your current OS, environment, and Python version "
+                    "we highly recommend you to read the update docs at <{docs}> and "
+                    "make sure there is nothing else that "
+                    "needs to be done during the update. "
+                    "**Should the above not work for you,"
+                    " ensure you have read the update docs.**"
```

Comment:
I think this sentence doesn't fit into this string after it got rewritten. I think it should just be removed and rest of the string should be emphasised (or maybe only some part of it? not sure)

Reply:
I think just rewording it to say "should the below not work..." I'm not in favour of emboldening a whole paragraph

Reply:
I don't know, it seems kind of out of place, maybe we could just bold the part about docs?
I.e.
```
While the following should work in most scenarios as it is based on your current OS, environment, and Python version, **we highly recommend you to read the update docs at <{docs}> and make sure there is nothing else that needs to be done during the update.**
```

Reply:
Can you make a suggestion please on my phone